libs/cardano-ledger-test/bench/Bench/Cardano/Ledger/EpochBoundary.hs

```diff
@@ -87,13 +96,17 @@ txOutsFromPtrs ptrs =
     coinVal = fromJust . toCompact . Val.inject $ Coin 200
 
 -- | Generate n stake credentials
-stakeCreds :: Word64 -> [StakeCredential TestCrypto]
-stakeCreds n =
-  [0 .. n]
-    <&> (\i -> KeyHashObj (hashKey . VKey $ VerKeyMockDSIGN i))
+stakeCreds :: Word64 -> [StakeCredential]
+stakeCreds n = replicate (fromIntegral n) $ KeyHashObj stakeAddr28
```

Comment:
This isn't gonna work, because that will produce identical credentials, where we need distinct

Reply:
Fixed it